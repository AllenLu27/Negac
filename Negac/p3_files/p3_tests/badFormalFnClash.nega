bool foo(){ }
void main(void foo){ }
