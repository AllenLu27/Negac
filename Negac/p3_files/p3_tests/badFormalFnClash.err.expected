FATAL [2,16]: Invalid type in declaration
Semantic analysis failed
